libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -37,6 +37,9 @@ class AP_Airspeed
     // constructor
     AP_Airspeed();
 
+void set_fixedwing_parameters(const class AP_FixedWing *_fixed_wing_parameters);
```

Comment:
Just needs to be spaced in for consistent whitespace.

Reply:
done